scully/renderPlugins/puppeteerRenderPlugin.ts

```diff
@@ -9,10 +9,14 @@ import {logError, yellow} from '../utils/log';
 import {launchedBrowser} from './launchedBrowser';
 
 export const puppeteerRender = async (route: HandledRoute): Promise<string> => {
-  // const {version} = JSON.parse(readFileSync(join('../package.json')).toString()) || '0.0.0';
-  const path = scullyConfig.hostUrl === null ?
-    `http://localhost:${scullyConfig.appPort}${route.route}`
-    : `${scullyConfig.hostUrl}${route.route}`;
+  let version = '0.0.0';
+  try {
+    const {version: pkgVersion} = JSON.parse(readFileSync(join('../package.json')).toString()) || '0.0.0';
+    version = pkgVersion;
+  } catch {}
+  const path = scullyConfig.hostUrl
+    ? `${scullyConfig.hostUrl}${route.route}`
+    : `http://${scullyConfig.hostName}:${scullyConfig.appPort}${route.route}`;
```

Comment:
@SanderElias with Aaron we did several tests with this code and we left the other one due to errors, for thjs, we wrote this code
```const path = scullyConfig.hostUrl === null ?
    `http://localhost:${scullyConfig.appPort}${route.route}`
    : `${scullyConfig.hostUrl}${route.route}`;```

Reply:
If localhost not working in some computer you can use the hostUrl for put your 0.0.0.0 or whatelse you need

Reply:
those are 2 distinct problems.
1. need to connect to a 3rth party server. (`hostUrl`)
2. don't use `localhost` because of system limitations/preferences. (hostName)
the `hostUrl` doesn't change the local bind-to-ipAddress problem.
the `hostName` doesn't connect to a 3rth party